ethicalads-theme/templates/ea/homepage.html

```diff
@@ -108,8 +107,8 @@ <h3>Respectful</h3>
     <div class="row justify-content-center">
 
       <div class="col-md-6 mb-3">
-        <h2 class="font-weight-bold mb-8">One tasteful ad</h2>
-        <p class="font-size-lg text-gray-800">We focus exclusively on advertising to developers by showing a single high quality ad on the sites they visit. We allow topic targeting like <strong>data science</strong> or <strong>web development</strong>. Our privacy-preserving targeting gets great results for marketers and more money for publishers.</p>
+        <h2 class="font-weight-bold mb-8">High-performing developer ads</h2>
+        <p class="font-size-lg text-gray-800">Reach developers on high-quality sites and connect with your exact tech audience whether that is data scientists, security engineers, or web developers.</p>
```

Comment:
```suggestion
        <p class="font-size-lg text-gray-800">Reach developers on high-quality sites and connect with your exact tech audience.</p>
```

Not sure the list buys us much here, probably better to have a more exhaustive UI list, or not list it.